### PR TITLE
Updated Assurance Case 2 Explanation

### LINKED ISSUE #37: Assessment of evidence for Assurance case 2
Assess the alignment of the evidence you identified in your diagrams, with that available (or can be made available) from the OSS project. Highlight the gaps.

### PR DESCRIPTION
**Pull request by CernickyR**

### CODE REVIEW

File: Assurance Cases.md [hunk 1]
```diff
@@ -35,17 +35,52 @@ The final gap that I noticed was with evidence E6. [Fail2Ban]( https://pimylifeu
 #### [Return to top](#assurance-case-list)
 
 ### Assurance Case 2
-#### .red[TODO] Insert Assurance Case 2 Name
+#### Home Assistant provides multiple Authentication Provders
```
Comment (Jbibianochavez) [inline Assurance Cases.md]: Misspelled provider here.

File: Assurance Cases.md [hunk 2]
```diff
@@ -35,17 +35,52 @@ The final gap that I noticed was with evidence E6. [Fail2Ban]( https://pimylifeu
...
+
+Home Assistant also allows Multi-Factor Authentication to be configured for the Home Assistant server authentication. As seen in R4, if the Multi Factor Authentication is compromised an advisary can exploit this to gain access to the server. E6 though states that Multi Factor Authentication is solely third party (such as Google Authenticatior or Authy) which is not managaed by Home Assistant.
+
+With Home assistant providing multiple ways for authentication it might provide multiple avenues for advisarys to attack the system as seen in R3. Home Assistant though by default only turns on the authentication prviders that you specify in then `configuration.yaml`. Any authentication providers that are not in said `configuration.yaml`, are truned off and no able to be used as seen in E5.
```
Comment (Jbibianochavez) [inline Assurance Cases.md:53]: Misspelled providers.

File: Assurance Cases.md (hunk 1 above)
Comment (CernickyR) [reply Assurance Cases.md]: updated

File: Assurance Cases.md (hunk 2 above)
Comment (CernickyR) [reply Assurance Cases.md:53]: updated

**Review by Jbibianochavez — APPROVED** [review]
Looks good!

**Review by RocketSockz — APPROVED** [review]
looks go to me!

**Pull request merged**